pkg/api/v1/load_balancers_create.go

```diff
@@ -32,63 +34,69 @@ func (r *Router) loadBalancerCreate(c echo.Context) error {
 		return v1BadRequestResponse(c, err)
 	}
 
-	lbs := models.LoadBalancerSlice{}
-
 	tx, err := r.db.BeginTx(ctx, nil)
 	if err != nil {
 		r.logger.Errorw("failed to begin transaction", "error", err)
 		return v1BadRequestResponse(c, err)
 	}
 
-	for _, p := range payload {
-		lb := &models.LoadBalancer{
-			TenantID:         tenantID,
-			DisplayName:      p.DisplayName,
-			LoadBalancerSize: p.LoadBalancerSize,
-			LoadBalancerType: p.LoadBalancerType,
-			IPAddr:           p.IPAddr,
-			LocationID:       p.LocationID,
-			Slug:             slug.Make(p.DisplayName),
-			CurrentState:     "provisioning",
-		}
-
-		if err := validateLoadBalancer(lb); err != nil {
-			_ = tx.Rollback()
-
-			r.logger.Errorw("failed to validate load balancer", "error", err)
+	lb := &models.LoadBalancer{
+		TenantID:         tenantID,
+		DisplayName:      payload.DisplayName,
+		LoadBalancerSize: payload.LoadBalancerSize,
+		LoadBalancerType: payload.LoadBalancerType,
+		IPAddr:           payload.IPAddr,
+		LocationID:       payload.LocationID,
+		Slug:             slug.Make(payload.DisplayName),
+		CurrentState:     "provisioning",
```

Comment:
Im curious if you all talked about a "requested" state?  technically it's not provisioning yet

Reply:
Yes. This is more or less a stub until `status` is introduced and extends this to multiple status' that apply to the lifecycle mgmt of a LB instance